apps/tutorial-servo-can-control-steven/main.cpp

```diff
@@ -0,0 +1,24 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "CANMsg.h"
+#include "TutorialServo.h"
+#include "mbed.h"
+
+// Declare global variables
+CAN can(CAN1_RX, CAN1_TX);
```

Comment:
FW uses a custom `CANBus` wrapper class instead of mbed's `CAN` API directly. Not sure if it's required for the training, but it would probably be better to use that instead. See the arm app for an example.